redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -151,6 +139,25 @@ public function send()
         });
     }
 
+    /**
+     * @return void
+     */
+    protected function prepareDetourMode($addon)
```

Comment:
```suggestion
    private function prepareDetourMode(): void
```

* Bei neuen Methoden gerne direkt native Typen  nutzen (`: void`)
* Hast du die bewusst protected gesetzt? Ansonsten gilt bei uns, dass neue Methoden default private sein sollten, und nur bei echtem Bedarf auf protected oder public gesetzt werden
* Der Addon-Param scheint nun ja doch nicht genutzt zu werden, da die Methode sich selbst das Addon holt. Oder alternativ doch den Param behalten, dann aber in der Methode auch damit arbeiten und nicht das Addon-objekt neu holen (beide Varianten für mich ok).

Reply:
ok. Alles klar. Ich wollte $addon zentral holen. Aber das wollte irgendwie nicht vorher. Dann habe ich vergessen es wieder rauszunehmen. 
Danke für die Hinweise. Werde mich dran halten. War mir nicht sicher, wie wir aktuell im core damit umgehen.